dallinger/prolific.py

```diff
@@ -281,3 +281,91 @@ def _req(self, method: str, endpoint: str, **kw) -> dict:
             raise ProlificServiceException(json.dumps(error))
 
         return parsed
+
+
+class DevProlificService(ProlificService):
+    """Wrapper that mocks the Prolific REST API and instead of making requests it writes to the log."""
+
+    def __init__(self, api_token: str, api_version: str, referer_header: str):
```

Comment:
This overriding of `__init__` seems redundant, consider deleting

Reply:
Yep, done.